instance/static/js/src/instance.js

```diff
@@ -216,6 +248,8 @@ app.controller("Details", ['$scope', '$state', '$stateParams', 'OpenCraftAPI',
                     $scope.is_spawning_appserver = false;
                 }
                 $scope.old_appserver_count = instance.appserver_count;
+                $scope.originalNotes = $scope.instance.notes;
+                $scope.renderMarkDownElements();
```

Comment:
@fferrin, can we take a more functional approach here - something like 

```
<div ng-if="!isEditingNotes" class="markdown">{{ render(instance.notes) }}</div>
```

```
render = (text) => dompurify.sanitize(marked.parse(text))
```

Reply:
@swalladge That looks better by the way, but I think we should either set the string to the `innerHTML` attribute or create a new child, right?

Reply:
Yep, I'm not that familiar with how angular does it, but I'm sure it has a method to output the raw html instead of escaping it. We want to find a function that lets angular handle it; we don't want to maintain other functions that manually set the innerhtml.

Reply:
I'll look for it then